### PR TITLE
Merge 8.1.x into develop

### PR DESCRIPTION
**Pull request by nicosomb**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Merge 8.1.x into develop
| Type?             | improvement 
| Category?         | ME
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI and auto tests 🟢
| Fixed ticket?     | ~
| Related PRs       | ~
| Sponsor company   | PrestaShop SA

### CODE REVIEW

**Comment by nicosomb** [comment]
Auto tests here https://github.com/nicosomb/ga.tests.ui.pr/actions/runs/4260678177

File: classes/Link.php [hunk 1]
```diff
@@ -1027,22 +1028,23 @@ public function getImageLink($name, $idImage, $type = null, string $extension =
      * Returns a link to a supplier image for display.
      *
      * @param int $idSupplier
-     * @param string|null $type image type (small_default, medium_default, large_default, etc.)
+     * @param string|null $type Image thumbnail name (small_default, medium_default, large_default, etc.)
```
Comment (nicosomb) [inline classes/Link.php:1031]: @Hlavtox as I had conflicts here, I want to have your review to be sure I didn't make mistake.

**Comment by nicosomb** [comment]
@FabienPapet @zuk3975 as I had conflicts in `src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml`, I want to have your review to be sure I didn't make mistake.

File: classes/Link.php (hunk 1 above)
Comment (Hlavtox) [reply classes/Link.php:1031]: @nicosomb It doesn't matter really, but go with the new one. :-)

**Comment by nicosomb** [comment]
New auto tests https://github.com/nicosomb/ga.tests.ui.pr/actions/runs/4262140047

**Review by matthieu-rolland — APPROVED** [review]

**Review by FabienPapet — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]
:heavy_check_mark: Reviewed & Approved for `tests/UI` modifications

**Review by boubkerbribri — APPROVED** [review]
LGTM, waiting for Auto tests to finish

**Review by Progi1984 — CHANGES_REQUESTED** [review]
I check the automated tests : On `tests/UI/campaigns/functional/BO/11_international/01_localization/03_currencies/05_filterSortAndPagination.ts`, it fails with this log 

```
[2023-02-27 16:57:21] php.CRITICAL: Fatal Compile Error: ContainerDkxp9So\appAppKernelDevDebugContainer::load(): Failed opening required '/projects/prestashop/var/cache/dev/ContainerDkxp9So/getPropertyInfo_SerializerExtractorService.php' (include_path='/projects/prestashop/vendor/pear/archive_tar:/home/flefevre/projects/prestashop/vendor/pear/console_getopt:/projects/prestashop/vendor/pear/pear-core-minimal/src:/projects/prestashop/vendor/pear/pear_exception:.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Compile Error: ContainerDkxp9So\\appAppKernelDevDebugContainer::load(): Failed opening required '/projects/prestashop/var/cache/dev/ContainerDkxp9So/getPropertyInfo_SerializerExtractorService.php' (include_path='/projects/prestashop/vendor/pear/archive_tar:/projects/prestashop/vendor/pear/console_getopt:/projects/prestashop/vendor/pear/pear-core-minimal/src:/projects/prestashop/vendor/pear/pear_exception:.:/usr/share/php') at /projects/prestashop/var/cache/dev/ContainerDkxp9So/appAppKernelDevDebugContainer.php:2057)"} []
```

**Review by Progi1984 — APPROVED** [review]

**Comment by nicosomb** [comment]
Thank you everyone.

**Pull request merged**